gui/src/i18n/fr/translation.json

```diff
@@ -80,16 +80,16 @@
   },
   "overlay": {
     "is-visible-label": "Superposer le squelette dans SteamVR",
-    "is-mirrored-label": "Afficher le squelette en miroir"
+    "is-mirrored-label": "Afficher le squelette en tant que miroir"
   },
   "tracker": {
     "status": {
       "none": "Pas de statut",
       "busy": "Occupé",
       "error": "Erreur",
       "disconnected": "Déconnecté",
-      "occluded": "Occulté",
-      "ok": "Ok"
+      "occluded": "Obstrué",
+      "ok": "Connecté"
```

Comment:
What's your thoughts behind this? A tracker can be connected but occluded... If you want something more french than "Ok", maybe "Correcte" would work? (Not 100% against Connecté tho!)

Reply:
Well, I simply followed english's translation.json

```json
      "disconnected": "Disconnected",
      "occluded": "Occluded",
      "ok": "Connected"
    },
```

Reply:
Uh didn't remember english was "Connected"... "ok" is actually for the SteamVR TrackerStatus.OK...
Ig "Connecté" is fine after all